src/native/libs/System.Security.Cryptography.Native/pal_x509.c

```diff
@@ -452,6 +452,11 @@ static DIR* OpenUserStore(const char* storePath, char** pathTmp, size_t* pathTmp
     // Leave one byte for '\0' and one for '/'
     size_t allocSize = storePathLen + sizeof(ent->d_name) + 2;
     char* tmp = (char*)calloc(allocSize, sizeof(char));
+    if (!tmp)
+    {
+        return NULL;
```

Comment:
The other path in this method that returns null also sets `pathTmp` and `nextFileWrite`. We should be consistent.

Reply:
Fixed. Thank you.